### DISCUSSION TITLE
Timeouts on raspPi and on ubuntu VM , runs fine on Windows 10

### DISCUSSION THREAD

**Discussion by veetek22** [jef/streetmerchant#1206] [Help]
I'm getting timeouts on ubuntu , tried increasing the page timeout to 50000 but that didn't help.

I ran it on windows and it's working fine on older intel I5 now  raspberry pi only gets timeouts nothing else , I also ran it in VM on freenas which hhas a xeon E3-1220V5 and getting like 70% timeouts.

Is anyone running it succesfuly on ubuntu ?

Thank you,

**Comment by baddino** [comment]
same problem :(

**Comment by baddino** [comment]
> I'm getting timeouts on ubuntu , tried increasing the page timeout to 50000 but that didn't help.
> 
> I ran it on windows and it's working fine on older intel I5 now raspberry pi only gets timeouts nothing else , I also ran it in VM on freenas which hhas a xeon E3-1220V5 and getting like 70% timeouts.
> 
> Is anyone running it succesfuly on ubuntu ?
> 
> Thank you,

I have solved with set on .env file -> PAGE_TIMEOUT="0"

**Comment by veetek22** [comment]
So it looks like when you set this to 0 you no longer get the timeouts but it is soooo slow to get results
